docs/manual/common/concepts/InternalAndExternalCommunication.md

```diff
@@ -1,6 +1,6 @@
 #Internal and external communication
 
-Lagom promotes use of asynchronous communication without preventing use of synchronous communication where necessary. For example, as mentioned in [[Polyglot systems with Lagom|PolyglotSystems]], legacy application components might use synchronous communication that cannot be changed.  When choosing the mode of communication for your microservices, keep in mind inter-service communication as well as external communication. 
+Lagom promotes use of asynchronous communication without preventing use of synchronous communication where necessary. For example, as mentioned in [[Polyglot systems with Lagom|PolyglotSystems]], legacy application components might use synchronous communication that cannot be changed. Within your system, services might communicate with other services in the same node and services in other nodes.  
```

Comment:
The new last sentence is technically true, but I don't think is the meaning we want to capture. There's no difference between communicating with other services on the same node vs. other nodes.

The more important distinction is communicating between other nodes of the _same_ service vs. communicating with other services.

Like this:

![lagom communication 001](https://cloud.githubusercontent.com/assets/44385/24383224/633d38b0-13a7-11e7-8514-2e292c54eb79.png)